Tests/NIOTests/BootstrapTest.swift

```diff
@@ -529,6 +529,322 @@ class BootstrapTest: XCTestCase {
         XCTAssertNil(NIOPipeBootstrap(validatingGroup: elg))
         XCTAssertNil(NIOPipeBootstrap(validatingGroup: el))
     }
+    
+    func testShorthandServerOptionsAreEquivalent() throws {
+        func bindAndGetOption<Option>(option: Option, _ applyBootstrapOptions: (ServerBootstrap) -> ServerBootstrap) throws ->
+                Option.Value
+                where Option : ChannelOption {
+            let bootstrap = applyBootstrapOptions(ServerBootstrap(group: group))
+            let serverChannel = try bootstrap.bind(host: "127.0.0.1", port: 0).wait()
+            let optionValue = try serverChannel.getOption(option).wait()
+            try serverChannel.close().wait()
+            return optionValue
+        }
+        
+        func checkOptionEquivalence<Option>(longOption: Option, setValue: Option.Value,
+                                            shortOption: ServerBootstrap.ServerOption) throws
+            where Option : ChannelOption, Option.Value : Equatable {
+            let longSetValue = try bindAndGetOption(
+                option: longOption, { bs in
+                bs.serverChannelOption(longOption, value: setValue) })
```

Comment:
This formatting is pretty wild! One thing to note is that Swift has some nice syntactic sugar in that it allows the last closure of a function call to be after the closing parentheses ("trailing closure syntax"). Your call here would become the following:

```swift
let logSetValue = try bindAndGetOption(option: longOption) { bs in
    bs.serverChannelOption(longOption, value: setValue)
}
```

Reply:
I have adopted this syntax.